packages/xrpl/src/models/methods/serverState.ts

```diff
@@ -48,24 +48,30 @@ export interface ServerStateResponse extends BaseResponse {
       load_factor_fee_queue?: number
       load_factor_fee_reference?: number
       load_factor_server?: number
+      peer_disconnects?: string
+      peer_disconnects_resources?: string
       peers: number
       pubkey_node: string
       pubkey_validator?: string
       server_state: ServerState
       server_state_duration_us: number
-      state_accounting: Record<ServerState, StateAccounting>
+      // The distinction between full, validating, and proposing is based on synchronization with the rest of the global network,
+      // and it is normal for a server to fluctuate between these states as a course of general operation.
+      // Construct a type that requires at least one of these fields to be present.
+      state_accounting: StateAccountingFinal
       time: string
       uptime: number
       validated_ledger?: {
-        age: number
+        age?: number
         base_fee: number
+        close_time: number
         hash: string
         reserve_base: number
         reserve_inc: number
         seq: number
       }
       validation_quorum: number
-      validator_list_expires?: string
+      validator_list_expires?: number
```

Comment:
If anything is incorrect in the xrpl.org docs, there should be an issue opened there to fix it.

Reply:
It has since been added https://github.com/XRPLF/xrpl-dev-portal/blame/6833f0c03947263d916a7b598980d9627867f479/content/references/http-websocket-apis/public-api-methods/server-info-methods/server_state.md#L311